bin/plugin/commands/performance.js

```diff
@@ -328,15 +314,39 @@ async function runPerformanceTests( branches, options ) {
 			buildPath
 		);
 
-		await runShellScript(
-			'cp ' +
-				path.resolve(
-					performanceTestDirectory,
-					'bin/plugin/utils/.wp-env.performance.json'
-				) +
-				' ' +
-				environmentDirectory +
-				'/.wp-env.json'
+		// Create the config file for the current env.
+		fs.writeFileSync(
+			path.join( environmentDirectory, '.wp-env.json' ),
```

Comment:
similar question here about removing `performance` from the name. when I download these files they lose some inherent linkage to the performance CI job that created them.

Reply:
[It was the same before](https://github.com/WordPress/gutenberg/blob/699dd44767d4b77fb5499fc764e093057b01ced5/bin/plugin/commands/performance.js#L331-L340) - we were copying the `.wp-env.performance.json` and renaming it to `.wp-env.json`, so in this regard nothing has changed. I assumed the default `.wp-env.json` name is used for the `wp-env start` script to pick it up without any extra arguments.